pages/news-and-events/index.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="news-and-events-page">
+    <breadcrumb-trail :breadcrumb="breadcrumb" :title="pageTitle.toUpperCase()" />
+    <div :style="urlStyle" class="news-events-banner">
+      <div class="banner__title-box">
+        <span>{{banner.values.title}}</span>
+      </div>
+    </div>
+    <LatestNews :newsList="topNews.newsList"/>
```

Comment:
Change LatestNews to latest-news please.

One of the least clear rules about the naming is to turn the name into kebab-case.
The component should be named as PascalCase but it should be kebab-case when used in the template. Vue would interpreted them automatically.